### PR TITLE
GPU MisconfigurationException fix

### PR DESCRIPTION
**Pull request by adrianlachowicz**

### CODE REVIEW

File: train.py [hunk 1]
```diff
@@ -118,7 +118,7 @@ def main(args):
     # set GPU to use
     if args.gpu > -1:
         os.environ["CUDA_DEVICE_ORDER"] = "PCI_BUS_ID"
-        os.environ["CUDA_VISIBLE_DEVICES"] = str(args.gpu)
+        #os.environ["CUDA_VISIBLE_DEVICES"] = str(args.gpu)
```
Comment (majsylw) [inline train.py:121]: I think we can just remove that line ;-)

**Review by majsylw — APPROVED** [review]

File: train.py (hunk 1 above)
Comment (adrianlachowicz) [reply train.py]: Ok, removed :)

**Review by alicjak1519 — APPROVED** [review]
Wow, I wasn't aware that this line can cause problems. Great work. :)

File: train.py (hunk 1 above)
Comment (AgaMiko) [reply train.py]: No we cannot! This is the only method to make sure that ONLY one card will be reserved. This is NOT a BUG!

**Comment by AgaMiko** [comment]
This changes has to be reversed. It might break out pipeline! This is the only method that always force the model to reserve only one card. GPU parameters can be changes only using argument parser.

**Pull request merged**